sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/DeltaKey.java

```diff
@@ -0,0 +1,49 @@
+package com.bazaarvoice.emodb.sor.db.astyanax;
+
+import com.netflix.astyanax.annotations.Component;
+
+import java.util.UUID;
+
+public class DeltaKey {
+
+    private @Component(ordinal=0) UUID _changeId;
+    private @Component(ordinal=1) Integer _block;
```

Comment:
i think it's best to leave it as an object because that way we don't have to worry about the int default value of 0 accidentally being compared an actual 0 block (i.e. the first block in a delta)